pkg/controller/account/account_controller.go

```diff
@@ -344,6 +344,11 @@ func (r *ReconcileAccount) Reconcile(request reconcile.Request) (reconcile.Resul
 	}
 	regionAMIs := processConfigMapRegions(stringRegions)
 
+	awsAssumedRoleClient, creds, err := r.handleCreateAdminAccessRole(reqLogger, currentAcctInstance, awsSetupClient)
+	if err != nil {
+		return reconcile.Result{}, err
+	}
+
```

Comment:
Is there a reason this got moved up?  This seems to be breaking STS cluster initialization.

Reply:
Ahh, figured it out.  Patch included in review comment.